### PR TITLE
Fixes CLI - hammer user list now contains more fields

### PR DESCRIPTION
**Pull request by ntkathole**
Issue #5324 

Test results

```
# pytest tests/foreman/cli/test_user.py -k test_positive_list
========================================================== test session starts ==========================================================
collected 51 items

tests/foreman/cli/test_user.py ....

========================================================== 47 tests deselected ==========================================================
=============================================== 4 passed, 47 deselected in 397.97 seconds ===========================================
```

### CODE REVIEW

**Comment by ldjebran** [comment]
Do not think that the purpose of this tests, to test the content of all listed fields, but perhaps to verify the only needed ones. 

If we need to verify that all fields are returned we have to create one test for that purpose.

@oshtaier @abalakh what do you think?

* We will face this tests to fail every time a field is added/removed

**Comment by abalakh** [comment]
@ldjebran completely agree with you. Even inline comment says `# make sure user is in list result`.
We should only check result[0] is our user, and should not compare every possible field. I believe checking id and login should be enough.

**Comment by ntkathole** [comment]
@abalakh and @ldjebran Thanks...I am also agree with your comments...will update the PR accordingly.

**Comment by ntkathole** [comment]
@ldjebran and @abalakh Changed some logic during assert...I found this shortest solution...please suggest if this could be more better.Thanks :)

File: tests/foreman/cli/test_user.py [hunk 1]
```diff
@@ -527,11 +527,10 @@ def test_positive_list_username(self):
                 self.assertEqual(len(result), 1)
                 # make sure user is in list result
                 self.assertEqual({
-                    'email': user['email'],
                     'id': user['id'],
                     'login': user['login'],
-                    'name': user['name'],
-                }, result[0])
+                }, {k: v for k, v in result[0].items()
+                    if k in ['id', 'login']})
```
Comment (abalakh) [inline tests/foreman/cli/test_user.py]: Why not simply
```python
self.assertEqual(
    {user['id'], user['login']},
    {result[0]['id'], result[0]['login']}
)
```
we're interested in values, not keys.

**Comment by ntkathole** [comment]
@abalakh @ldjebran comment addressed!

**Pull request merged**